_content/setup-wallets-commandline.md

```diff
@@ -0,0 +1,141 @@
+## Interledger Protocol (ILP) TestNet: Getting Started
+
+This tutorial describes how to use the command line and test XRP to:
+
+1. Create an account in the Xpring.io Interledger Testnet.
+2. Add test XRP to your account using the TestNet Rainmaker \(our version of a "faucet"\).
+3. Check your account balance.
+4. Send test XRP to another payment pointer.
+5. Receive test XRP.
+
+To do the same with a GUI-based interface, refer to [setup-wallets.md].
+
+### 1. Create an account
+
+Create a new account with this command:
+
+```
+> curl --location --request POST 'https://hermes-rest.ilpv4.dev/accounts' \
```

Comment:
Hopefully we'll have new URLs here in the next day or two. @LoisRP can you check with me before merging this?

Reply:
Will do. Incidentally, I don't have access to this repo: https://github.com/xpring-eng/hermes-ilp/ .

Reply:
Confirmed we keep the same URL for now (until a week or two from now).